recipe/meta.yaml

```diff
@@ -20,8 +20,8 @@ source:
     - 0102-include-cc-in-pip-package.patch
     - 0103-BUILD-changes-for-external-sqlite-package.patch
     - 0104-Fixed-linking-error-in-conda-builds.patch
-    - 0105-Changed-absl-flat_hash_set-initialization.patch
-    - 0106-Upstream-patch-for-absl-error.patch    #[x86_64] 
+    - 0105-Eigen-customized.patch                         #[ppc64le]
```

Comment:
Let's move this patch to the 3xx section (temporary fix to a problem) as presumably this will be fixed upstream at some point.